### PR TITLE
Fix build:example handling

### PR DESCRIPTION
**Pull request by blink1073**

### CODE REVIEW

File: example/package.json [hunk 1]
```diff
@@ -7,6 +7,7 @@
     "phosphor-commandpalette": "file:.."
   },
   "scripts": {
+    "build": "cd .. && npm run build:example",
```
Comment (sccolbert) [inline example/package.json]: :-1: on this line. I consider this private `package.json` an implementation detail.

File: package.json [hunk 2]
```diff
@@ -33,8 +33,8 @@
   },
   "scripts": {
     "clean": "rimraf lib",
-    "clean:example": "rimraf example/build && rimraf example/node_modules/phosphor-commandpalette",
-    "build:example": "cd example && npm install && tsc && webpack --config ./webpack.conf.js",
+    "clean:example": "rimraf example/build",
```
Comment (sccolbert) [inline package.json]: what about combining this with the `build:example` command too?

Comment (blink1073) [inline package.json]: Done.

File: example/package.json (hunk 1 above)
Comment (blink1073) [inline example/package.json]: This is useful to be able to build while editing the example.

**Pull request merged**